### PR TITLE
rename vampires to cult in lang file

### PR DESCRIPTION
**Pull request by ItsSammyM**
prophet
diciple
zealot

if these names are PC enough and descriptive enough this should be good
We should probably rename the types and variable names before merging this

### CODE REVIEW

File: client/src/resources/lang/en_us.json [hunk 1]
```diff
@@ -478,28 +478,28 @@
...
-    "chatMessage.draculaCanConvertTonight": "Dracula can convert tonight.",
-    "chatMessage.draculaCantConvertTonight": "Dracula can not convert tonight.",
-    "chatMessage.vampiresSacrificesRequired": "Dracula requires \\0 more sacrifices to convert.",
-    "chatMessage.vampiresSacrificesRequired.1": "Dracula requires 1 more sacrifice to convert.",
-    "chatMessage.vampiresSacrificesRequired.0": "Dracula has enough sacrifices to convert.",
+    "chatMessage.draculaCanConvertTonight": "Prophet can convert tonight.",
+    "chatMessage.draculaCantConvertTonight": "Prophet can not convert tonight.",
+    "chatMessage.vampiresSacrificesRequired": "Prophet requires \\0 more sacrifices to convert.",
+    "chatMessage.vampiresSacrificesRequired.1": "Prophet requires 1 more sacrifice to convert.",
+    "chatMessage.vampiresSacrificesRequired.0": "Prophet has enough sacrifices to convert.",
```
Comment (Jack-Papel) [inline client/src/resources/lang/en_us.json:485]: Should be "The Prophet", not just "Prophet"

File: client/src/resources/lang/en_us.json [hunk 2]
```diff
@@ -513,7 +513,7 @@
...
     "wiki.article.standard.convert.title": "Convert",
-    "wiki.article.standard.convert.text": "A player's role changes during the game. All of their ability uses are refilled to maximum. _For Example:_\n- If someone is converted to a Veteran, they get all three alerts.\n- A Vampire can convert players into vampires.",
+    "wiki.article.standard.convert.text": "A player's role changes during the game. All of their ability uses are refilled to maximum. _For Example:_\n- If someone is converted to a Veteran, they get all three alerts.\n- A Prophet can convert players into cultists.",
```
Comment (Jack-Papel) [inline client/src/resources/lang/en_us.json:516]: Can there be more than one? Should be "The Prophet" if not

File: client/src/resources/lang/en_us.json [hunk 3]
```diff
@@ -569,7 +569,7 @@
     "wiki.article.standard.outlineList.title":"Outline List",
     "wiki.article.standard.outlineList.text": "The outline list defines what roles are in the game. At the start of a game, each player gets assigned to a role outline. Then, their role is generated randomly from that role outline.",
     "wiki.article.standard.chatGroup.title": "Chat Group",
-    "wiki.article.standard.chatGroup.text": "Messages are sent to chat groups. The chat groups are:\n - All (no icon)\n - Dead 💀\n - Mafia 🌹\n - Vampire 🧛‍♂️\n - Haunt 👻\n - Jail ⛓️\n - Interview 📝\n\n Players can be in multiple chat groups at once, or even read chat groups they are not in.",
+    "wiki.article.standard.chatGroup.text": "Messages are sent to chat groups. The chat groups are:\n - All (no icon)\n - Dead 💀\n - Mafia 🌹\n - Cult 🪬\n - Haunt 👻\n - Jail ⛓️\n - Interview 📝\n\n Players can be in multiple chat groups at once, or even read chat groups they are not in.",
```
Comment (Jack-Papel) [inline client/src/resources/lang/en_us.json:572]: idk what this emoji is but my computer doesn't have it

File: client/src/resources/lang/en_us.json [hunk 4]
```diff
@@ -628,8 +628,8 @@
...
     "wiki.article.role.lookout.extra": "- You can not see astral visits\n- A framer can change your results",
 
-    "wiki.article.role.spy.guide": "Choose a player to find what happened to them. Each night you will passively learn who the mafia visited, weather or not dracula can convert, and how many vampires are alive. You should tell the town the information you get daily. Usually, the mafia can not visit themselves, so anyone who you see the maifa visits is likely not a mafia member.",
-    "wiki.article.role.spy.abilities":"* Visit a player to find out what happened to them\n* You will know who the mafia visited\n * At the start of night, you are told weather or not dracula can convert tonight\n* You will know how many vampires are alive",
+    "wiki.article.role.spy.guide": "Choose a player to find what happened to them. Each night you will passively learn who the mafia visited, weather or not prophet can convert, and how many cultists are alive. You should tell the town the information you get daily. Usually, the mafia can not visit themselves, so anyone who you see the maifa visits is likely not a mafia member.",
```
Comment (Jack-Papel) [inline client/src/resources/lang/en_us.json:631]: the prophet

File: client/src/resources/lang/en_us.json [hunk 5]
```diff
@@ -628,8 +628,8 @@
...
+    "wiki.article.role.spy.abilities":"* Visit a player to find out what happened to them\n* You will know who the mafia visited\n * At the start of night, you are told weather or not Prophet can convert tonight\n* You will know how many cultists are alive",
```
Comment (Jack-Papel) [inline client/src/resources/lang/en_us.json:632]: the Prophet

**Review by Jack-Papel — COMMENTED** [review]
Most useful review of all time
But yeah maybe we should change those variable names.
Tho tbh sometimes it's funny to see something named completely wrong in the code

File: client/src/resources/lang/en_us.json (hunk 3 above)
Comment (ItsSammyM) [reply client/src/resources/lang/en_us.json:572]: You should fix that. Its a really cool emoji

Comment (emt7553) [reply client/src/resources/lang/en_us.json:572]: Im fixing it dont touch

**Review by emt7553 — COMMENTED** [review]
Changed names across code and named roles: Apostle, Disciple, and Zealot

**Comment by Jack-Papel** [comment]
Will review after sam's changes

**Review by emt7553 — COMMENTED** [review]
Team/faction names changed to cult instead of cultists and wiki updated for cultists

File: client/src/resources/lang/en_us.json [hunk 6]
```diff
@@ -338,7 +338,7 @@
     "tag.dead":"💀",
 
     "chatGroup.mafia.icon": "🌹",
-    "chatGroup.vampire.icon": "🧛‍♂️",
+    "chatGroup.cult.icon": "🪬",
```
Comment (Jack-Papel) [inline client/src/resources/lang/en_us.json:341]: I don't have this emoji

File: client/src/resources/lang/en_us.json (hunk 3 above)
Comment (Jack-Papel) [reply client/src/resources/lang/en_us.json:572]: U did not fix it

File: server/src/game/chat/mod.rs [hunk 7]
```diff
@@ -115,9 +115,9 @@ pub enum ChatMessage {
     #[serde(rename_all = "camelCase")]
     PlayerWithNecronomicon{player_index: PlayerIndex},
     YourConvertFailed,
-    DraculaCanConvertTonight,
-    DraculaCantConvertTonight,
-    VampiresSacrificesRequired{required: u8},
+    ApostleCanConvertTonight,
+    ApostleCantConvertTonight,
+    cultSacrificesRequired{required: u8},
```
Comment (Jack-Papel) [inline server/src/game/chat/mod.rs:120]: Fix casing

File: server/src/game/role/wizard.rs [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+use crate::game::visit::Visit;
+use crate::game::Game;
+use crate::game::team::Team;
+use super::{Priority, RoleStateImpl};
+
+pub(super) const FACTION: Faction = Faction::None;
+pub(super) const MAXIMUM_COUNT: Option<u8> = None;
+
+#[derive(Clone, Debug, Serialize, Default)]
+pub struct Wizard{
```
Comment (Jack-Papel) [inline server/src/game/role/wizard.rs:16]: Why is this in this pull request? Seems unrelated.
Make another PR

Comment (Jack-Papel) [reply server/src/game/role/wizard.rs:16]: There is so much broken with this role, I'm not going to review it right now. Just remove it and make another pull request

**Review by Jack-Papel — CHANGES_REQUESTED** [review]
Why is Wizard in here? Should be another issue.
Wizard is also not implemented at all and it's completely broken.

**Comment by Jack-Papel** [comment]
@emt7553

**Review by emt7553 — COMMENTED** [review]
> Why is this in this pull request? Seems unrelated.
Make another PR 

This is not supposed to be on this branch, it was left over on my pc

**Review by emt7553 — COMMENTED** [review]
Removed accidental wizard role

**Review by Jack-Papel — APPROVED** [review]

File: client/src/resources/lang/en_us.json (hunk 2 above)
Comment (emt7553) [reply client/src/resources/lang/en_us.json]: Fixed
